i18n/en.json

```diff
@@ -5,7 +5,7 @@
     "gallery-loading-error-refresh": "Refresh",
     "gallery-loading-still": "Still loading",
     "gallery-unknown-author": "Author unknown",
-    "preview-loading-error": "There was an issue while displaying this preview.",
+    "preview-error-message": "There was an issue while displaying this preview.",
```

Comment:
You also need to rename the message in the qqq file.

Reply:
> Very good first pass. This is a well needed cleanup.
> 
> I think we can and should go further, see inline comments.

Great ideas from inline comments, thanks for the on point feedback. My interpretation of it now reflected in https://github.com/wikimedia/wikipedia-preview/pull/81/commits/0e014a1c411a9d3a5b6e37b1091388588db8f643, what do you think?